README.md

```diff
@@ -350,6 +350,36 @@ Future.reject('it broke')
 //> Left('it broke')
 ```
 
+#### finally
+##### `#finally :: Future a b ~> Future a c -> Future a b`
+##### `.finally :: Future a b -> Future a c -> Future a c`
```

Comment:
The functional API takes the "data" last, whereas in the method API `this` *is* the data.

So:

```js
computation.finally(finallyComputation)
==
Future.finally(finallyComputation, computation)
```

The different output type is meant to draw attention to that fact.